server/src/instant/db/permissioned_transaction.clj

```diff
@@ -158,31 +168,31 @@
     (object-delete-check ctx eid)))
 
 (defn object-checks
-  "Creates check commands for each object in the transaction. 
-  
-   We take tx-steps like: 
-   [ 
-     [:add-triple joe-eid :users/name \"Joe\"]  
+  "Creates check commands for each object in the transaction.
+
+   We take tx-steps like:
+   [
+     [:add-triple joe-eid :users/name \"Joe\"]
      [:add-triple joe-eid :users/age 32]
-     [:add-triple stopa-eid :users/name \"Stopa\"] 
+     [:add-triple stopa-eid :users/name \"Stopa\"]
      [:add-triple stopa-eid :users/age 30]
    ]
 
-   And we group them by `eid`. 
-   
+   And we group them by `eid`.
+
    {
-     joe-eid [[:add-triple joe-eid :users/name \"Joe\"]  
+     joe-eid [[:add-triple joe-eid :users/name \"Joe\"]
               [:add-triple joe-eid :users/age 32]]
-     stopa-eid [[:add-triple stopa-eid :users/name \"Stopa\"] 
-                [:add-triple stopa-eid :users/age 30]]  
+     stopa-eid [[:add-triple stopa-eid :users/name \"Stopa\"]
+                [:add-triple stopa-eid :users/age 30]]
    }
 
    With this, we can generate a grouped `check` command for each `eid`."
   [ctx tx-steps]
   (let [eid->tx-steps (group-by second tx-steps)
         eid-actions (mapcat ->eid-actions eid->tx-steps)]
     (->> eid-actions
-         (ua/vfuture-pmap (partial object-check ctx)))))
```

Comment:
No longer using `vfuture-pmap` because we shouldn't be doing any db fetches except for the preloads below. All of the work should be cpu-based.

Reply:
Beautiful!